hackathon_site/hackathon_site/utils.py

```diff
@@ -15,3 +18,18 @@ def is_registration_open():
     # is configured to match TIME_ZONE. We then make the datetime object timezone-aware.
     now = datetime.now().replace(tzinfo=settings.TZ_INFO)
     return settings.REGISTRATION_OPEN_DATE <= now < settings.REGISTRATION_CLOSE_DATE
+
+
+def formatDateTime(datetime, format):
+    """
+    Format a datetime using strftime. 
+    """
+    local_zone = tz.tzlocal()
+    # Replaces incompatible identifiers.
+    # Add to this function if there are any datetime format issues
+    if platform == "win32":
```

Comment:
We don't need to set it for MacOS right? because it is UNIX base. I am not sure.

Reply:
Nope! That's why it's only checking for windows.